conf/maestro-conf/edge-config-dell5000-demo.yaml

```diff
@@ -29,34 +31,38 @@ platform_readers:
 #     This is only a valid option if you are not using a well-known CA
 #    server_name: "devcloud.wigwag.io"
 var_defs:
+   - key: "SNAP_DATA"
+     value: "/var/snap/pelion-edge/current"
+   - key: "SNAP"
+     value: "/snap/pelion-edge/current"
    - key: "TMP_DIR"
      value: "/tmp"
    - key: "WIGWAG_NODE_PATH"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-core-modules/node_modules"
+     value: "{{SNAP}}/wigwag/devicejs-core-modules/node_modules"
    - key: "WIGWAG_DIR"
-     value: "/snap/pelion-edge/current/wigwag"
+     value: "{{SNAP}}/wigwag"
    - key: "NODE_EXEC"
-     value: "/snap/pelion-edge/current/usr/bin/node"
+     value: "{{SNAP}}/bin/node"
    - key: "DEVICEJS_ROOT"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-ng"
+     value: "{{SNAP}}/wigwag/devicejs-ng"
    - key: "DEVJS_CORE_MODULES"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-core-modules"
+     value: "{{SNAP}}/wigwag/devicejs-core-modules"
    - key: "MAESTRO_RUNNER_DIR"
-     value: "/snap/pelion-edge/current/wigwag/devicejs-core-modules/maestroRunner"
+     value: "{{SNAP}}/wigwag/devicejs-core-modules/maestroRunner"
```

Comment:
How do you know which variables need `{{SNAP}}`, and which need `{{SNAP_VAR}}`?

Reply:
Good question - The SNAP_VAR is writeable and SNAP is not. So, whenever we need to write something out we point it to SNAP_VAR.

Reply:
> The SNAP_VAR is writeable 

Slightly different wording, to make sure I understand.

The `SNAP_VAR` _directory_ is rw.
The `SNAP` _directory_ is ro.

Correct?

Reply:
That's correct, to be accurate its SNAP and SNAP_DATA. SNAP_DATA(which actually points to /var/snap) is rw and SNAP is ro.